### ISSUE TITLE
npm run build - multiple errors

### ISSUE THREAD

**Issue by simonefuoco** [webdriverio/expect-webdriverio#690]
After cloning the repository and running the command `npm install` all dependencies are installed.
At this point running the command `npm run build` give the following output:

```
> expect-webdriverio@3.1.4 build
> run-s clean compile

> expect-webdriverio@3.1.4 clean
> run-p clean:*

> expect-webdriverio@3.1.4 clean:tests
> rimraf test-types/**/node_modules && rimraf test-types/**/dist

> expect-webdriverio@3.1.4 clean:build
> rimraf ./lib

> expect-webdriverio@3.1.4 compile
> tsc --build tsconfig.build.json

src/types/expect-webdriverio.d.ts:1:1 - error TS1128: Declaration or statement expected.

1 ../../types/expect-webdriverio.d.ts
  ~

src/types/expect-webdriverio.d.ts:1:2 - error TS1128: Declaration or statement expected.

1 ../../types/expect-webdriverio.d.ts
   ~

Found 2 errors.

ERROR: "compile" exited with 1.
```

**Comment by simonefuoco** [comment]
I am ready to contribute to resolve the issue after discussing about it.

**Comment by christian-bromann** [comment]
@simonefuoco thanks for filing the issue. I was able to resolve the problem with this commit: https://github.com/webdriverio/expect-webdriverio/commit/2c8efefcce1b50b1b62abc7af1f1eed3d5cf7f9d

However I was not able to upgrade `@types/node` to latest as it would cause some typing issues with globals. I am not quite sure how to go about this. Any idea?

Closing as `main` branch should be green again.
